pkg/telemetery/empty_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package telemetery_test
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+)
+
+func TestDummy(*testing.T) {
```

Comment:
you can write a test for HCAdapter

Reply:
Yes, but I have to pass a Component.statusEvent to it, how do I create that?
Should I have to instantiate a Query server and monitor it?

Reply:
> I have to pass a Component.statusEvent to it, how do I create that?

you already created those events in the server when you were reporting Fatals etc.